src/github.com/getlantern/balancer/.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: go
+
+go:
+  - 1.3.3
```

Comment:
Oh yeah, most of our travis configs were already on 1.4, I just missed a few somehow.

Fixed - https://github.com/getlantern/flashlight-build/commit/a362d2cf49a8907b70217440c3f1a37ba885b4a8